server/api/users.js

```diff
@@ -12,3 +13,47 @@ router.get('/', (req, res, next) => {
     .then(users => res.json(users))
     .catch(next)
 })
+
+router.get('/:id', (req, res, next) => {
+  User.findOne({
+    where: {
+      id: req.params.id
+    },
+    include: [Order]
+  })
+    .then(user => res.json(user))
+    .catch(next)
+})
+
+router.put('/:id', (req, res, next) => {
+  User.update(req.body, {
+    where: {
+      id: req.params.id
+    },
+    returning: true
+  })
+   .then(user => res.status(200).json(user))
```

Comment:
User.update _does not return_ your updated user! It returns an array(?) with the number of rows affected. If you give it returning: true, it will _also_ return the updated user. You can either use .spread to get the user from the returned array or do .findById and then userInstance.update.